source/connect/stable-api.txt

```diff
@@ -0,0 +1,117 @@
+.. _php-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatible, backwards, upgrade
+
+.. note::
+
+   The {+stable-api+} feature requires {+mdb-server+} 5.0 or later.
+
+Overview
+--------
+
+In this guide, you can learn how to specify **{+stable-api+}** compatibility when 
+connecting to a MongoDB deployment.
+
+The {+stable-api+} feature forces the server to run operations with behaviors compatible 
+with the API version you specify. When you update either your library or server version, 
+the API version changes, which can change the way these operations behave.
+Using the {+stable-api+} ensures consistent responses from the server and 
+provides long-term API stability for your application.
+
+The following sections describe how you can enable and customize {+stable-api+} for
+your MongoDB client. For more information about the {+stable-api+}, including a list of 
+the commands it supports, see :manual:`Stable API </reference/stable-api/>` in the
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``MongoDB\Driver\ServerApi`` object and pass the {+stable-api+}
+   version you want to use. Currently, the library supports only version 1.
+#. Construct a ``MongoDB\Client`` object. For the ``driverOptions`` parameter, pass an
+   array that contains the ``serverApi`` connection option. Set this option to the
+   ``MongoDB\Driver\ServerApi`` object you created in the previous step.
+
+The following code example shows how to specify {+stable-api+} version 1:
+
+.. literalinclude:: /includes/connect/stable-api.php
+   :language: php
+   :copyable: true
+   :start-after: // start-specify-v1
+   :end-before: // end-specify-v1
+   :emphasize-lines: 5-7, 9
+
+.. note::
+
+   After you create a ``MongoDB\Client`` instance with
+   a specified API version, all commands you run with the client use the specified
+   version. If you need to run commands using more than one version of the 
+   {+stable-api+}, create a new ``MongoDB\Client`` instance.
+
+.. _stable-api-options:
+
+Configure the {+stable-api+}
+------------------------
+
+The following table describes the parameters of the ``MongoDB\Driver\ServerApi`` constructor.
+You can use these parameters to customize the behavior of the {+stable-api+}.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 25,75
+
+   * - Parameter
+     - Description
+
+   * - strict
+     - | **Optional**. When ``true``, if you call a command that isn't part of 
+         the declared API version, the library raises an exception.
+       |
+       | Default: ``false``
```

Comment:
I imagine you don't care to clarify this here, but the PHP extension actually defaults to `null`. It's the server that applies a `false` default (per [Stable API docs](https://www.mongodb.com/docs/manual/reference/stable-api/#parameters)).

I only bring this up as the [extension docs](https://www.php.net/manual/en/mongodb-driver-serverapi.construct.php) do correctly state:

> If no value is provided, the server default value (`false`) is used.

This applies to `deprecationErrors` below as well.

Reply:
reworded